src/components/molecules/search-results-list/results-list-item.jsx

```diff
@@ -30,9 +35,24 @@ const ResultsListItem = ({ result = {}, language = 'en' }) => {
 			);
 		}
 	}
+
+	const handleResultItemTitleClick = () => {
+		const pageUnit = urlQuery.get('pageunit');
+		tracking.trackEvent({
+			type: 'Other',
+			event: 'SitewideSearchApp:Other:ResultClick',
+			linkName: 'SiteWideSearchResults',
+			resultIndex: +itemIndex + 1 + +pageUnit,
```

Comment:
This calculation should be done outside the item in the results list.

Reply:
Additionally, it is unclear as to what numbers are what here.

Reply:
Updated! Moved computation outside of component.